docker/Dockerfile

```diff
@@ -1,45 +1,113 @@
-FROM clux/muslrust:1.70.0 as builder
-WORKDIR /app
-ARG CARGO_BUILD_TARGET=x86_64-unknown-linux-musl
-
-# comma-seperated list of features to enable
+ARG RUST_VERSION=1.71.0
+ARG ALPINE_VERSION=3.18
 ARG CARGO_BUILD_FEATURES=default
+ARG RUST_RELEASE_MODE=debug
+ARG UID=911
+ARG GID=911
+
```

Comment:
Would be good if you add a comment here to explain how cross-compile works. Or in a separate readme file.

Reply:
Yeah I'll add that